client/templates/register/register.scss

```diff
@@ -1,12 +1,3 @@
-.register {
```

Comment:
you know, this doesn't really have to go in right now, i'm just going to back it out.  we can figure it out later.